azureadgraph_connector.py

```diff
@@ -1097,7 +1107,10 @@ def _handle_validate_group(self, param):
         user_object_ids = list()
         for item in response.get('value', []):
             url = item.get('url', '')
-            match = re.match('https:\\/\\/graph.windows.net\\/{}\\/directoryObjects\\/(.+)\\/Microsoft.DirectoryServices.User$'.format(self._tenant), url)
```

Comment:
Update the regex such that it checks for the Base URL of the selected region, wherever applicable.

Reply:
done